public/doc/openapi-3-v1.2.json

```diff
@@ -3856,6 +3856,16 @@
             "example": "before",
             "description": "Denotes the scope in which the order item will run for the order it belongs to. It can be 'before', 'after', or 'applicable'",
             "readOnly": true
+          },
+          "artifacts": {
+            "type": "object",
+            "title": "Return values from a product provision",
+            "description": "Contains a key/value object that contains all of the information sent from a product provision",
+            "readOnly": true,
+            "example": {
+              "expose_to_cloud_redhat_com_important_id": 10,
```

Comment:
The prefix `expose_to_cloud_redhat_com_` has been stripped off. We should explain it in description also.